tools/render-test/slang-support.cpp

```diff
@@ -119,31 +119,18 @@ static const char computeEntryPointName[] = "computeMain";
         computeTranslationUnit = translationUnit;
     }
 
-
-    Slang::List<const char*> rawGlobalTypeNames;
-    for (auto typeName : request.globalGenericTypeArguments)
-        rawGlobalTypeNames.add(typeName.getBuffer());
-    spSetGlobalGenericArgs(
-        slangRequest,
-        (int)rawGlobalTypeNames.getCount(),
-        rawGlobalTypeNames.getBuffer());
-
-    Slang::List<const char*> rawEntryPointTypeNames;
-    for (auto typeName : request.entryPointGenericTypeArguments)
-        rawEntryPointTypeNames.add(typeName.getBuffer());
-
-    const int globalExistentialTypeCount = int(request.globalExistentialTypeArguments.getCount());
-    for(int ii = 0; ii < globalExistentialTypeCount; ++ii )
+    const int globalSpecializationArgCountCount = int(request.globalSpecializationArgs.getCount());
+    for(int ii = 0; ii < globalSpecializationArgCountCount; ++ii )
     {
-        spSetTypeNameForGlobalExistentialTypeParam(slangRequest, ii, request.globalExistentialTypeArguments[ii].getBuffer());
+        spSetTypeNameForGlobalExistentialTypeParam(slangRequest, ii, request.globalSpecializationArgs[ii].getBuffer());
     }
 
-    const int entryPointExistentialTypeCount = int(request.entryPointExistentialTypeArguments.getCount());
-    auto setEntryPointExistentialTypeArgs = [&](int entryPoint)
+    const int entryPointSpecializationArgCount = int(request.entryPointSpecializationArgs.getCount());
+    auto setEntryPointSpecializationArgs = [&](int entryPoint)
```

Comment:
FWIW that was already in the code when I started working on it (I don't recall introducing it), and at the time I didn't try to refactor things out to stand-alone `static` functions mostly because of my "keep the diffs small" rule.

I would be perfectly happy with a refactor to make those be `static` subroutines that make everything nice and clear.